Iterating over numbers, starting at 9 and ending at 3
9
8
7
6
5
4
3
While loop exited cleanly.
While should propagate exceptions out of its body.
And it does.
